src/index.js

```diff
@@ -3,7 +3,6 @@ export { default as Autocomplete } from "./Autocomplete";
 export { default as Checkbox } from "./Checkbox";
 export { default as Copyright } from "./Copyright";
 export { default as Color } from "./Color";
-export { default as DialogTitle } from "./DialogTitle";
```

Comment:
This should be added back. We want the DialogTitle to be a component that can be exported from this library and not just an internal component to the library.